openadapt/extensions/synchronized_queue.py

```diff
@@ -27,22 +27,28 @@ class SharedCounter(object):
     shared-counter-with-pythons-multiprocessing/
     """
 
-    def __init__(self, n=0):
-        self.count = multiprocessing.Value('i', n)
+    def __init__(self, n: int = 0) -> None:
+        """Initialize the shared counter.
+
+        Args:
+            n (int): The initial value of the counter. Defaults to 0.
+        """
+        self.count = multiprocessing.Value("i", n)
 
-    def increment(self, n=1):
-        """ Increment the counter by n (default = 1) """
+    def increment(self, n: int = 1) -> None:
+        """Increment the counter by n (default = 1)."""
         with self.count.get_lock():
             self.count.value += n
 
     @property
-    def value(self):
-        """ Return the value of the counter """
+    def value(self) -> Any:
```

Comment:
Should this be `int`?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507